### PR TITLE
Unmodifiable tag message. Error snackbar instead of dialog.

### PR DESCRIPTION
**Pull request by kejmil01**
TODO:
- tests

### CODE REVIEW

File: app/src/main/java/com/invisibleteam/goinvisible/mvvm/edition/EditActivity.java [hunk 1]
```diff
@@ -85,10 +87,14 @@ public static Intent buildIntent(Context context, ImageDetails imageDetails) {
     @VisibleForTesting(otherwise = VisibleForTesting.PRIVATE)
     final EditTagListener editTagListener = new EditTagListener() {
         @Override
-        public void openEditDialog(Tag tag) {
+        public void openTagEditionView(Tag tag) {
             if (tag.getKey().equals(ExifInterface.TAG_GPS_LATITUDE)) {
                 EditActivity.this.tag = tag;
                 startPlaceIntent();
+            } else if (tag.getTagType().getInputType().equals(InputType.UNMODIFIABLE)) {
```
Comment (michal1990) [inline app/src/main/java/com/invisibleteam/goinvisible/mvvm/edition/EditActivity.java]: better approach is comparing enum value with sth - we are securing against NUllPointerException

File: app/src/main/java/com/invisibleteam/goinvisible/mvvm/edition/EditActivity.java [hunk 2]
```diff
@@ -85,10 +87,14 @@ public static Intent buildIntent(Context context, ImageDetails imageDetails) {
     @VisibleForTesting(otherwise = VisibleForTesting.PRIVATE)
     final EditTagListener editTagListener = new EditTagListener() {
         @Override
-        public void openEditDialog(Tag tag) {
+        public void openTagEditionView(Tag tag) {
             if (tag.getKey().equals(ExifInterface.TAG_GPS_LATITUDE)) {
```
Comment (michal1990) [inline app/src/main/java/com/invisibleteam/goinvisible/mvvm/edition/EditActivity.java]: better approach is comparing constant value with sth - we are securing against NUllPointerException

File: app/src/main/java/com/invisibleteam/goinvisible/mvvm/edition/EditActivity.java [hunk 3]
```diff
@@ -85,10 +87,14 @@ public static Intent buildIntent(Context context, ImageDetails imageDetails) {
...
                 EditActivity.this.tag = tag;
                 startPlaceIntent();
+            } else if (tag.getTagType().getInputType().equals(InputType.UNMODIFIABLE)) {
+                showSnackBar(R.string.unmodifiable_tag_message);
+            } else if (tag.getTagType().getInputType().equals(InputType.INDEFINITE)) {
```
Comment (michal1990) [inline app/src/main/java/com/invisibleteam/goinvisible/mvvm/edition/EditActivity.java]: better approach is comparing enum value with sth - we are securing against NUllPointerException

File: app/src/main/java/com/invisibleteam/goinvisible/mvvm/edition/dialog/DialogFactory.java [hunk 4]
```diff
@@ -163,15 +160,4 @@ void validateText(TextDialogBinding binding,
         viewModel.setIsError(true);
         binding.valueTextLayout.setError("Error");
```
Comment (michal1990) [inline app/src/main/java/com/invisibleteam/goinvisible/mvvm/edition/dialog/DialogFactory.java]: by the occasion, can You move this string to res?

File: app/src/main/java/com/invisibleteam/goinvisible/mvvm/edition/EditActivity.java [hunk 5]
```diff
@@ -98,8 +104,20 @@ public void openEditDialog(Tag tag) {
...
             invalidateOptionsMenu();
         }
+
+        @Override
+        public void onEditError() {
+            showSnackBar(R.string.error_message);
+        }
     };
 
+    private void showSnackBar(int unmodifiable_tag_message) {
```
Comment (BartSoftware) [inline app/src/main/java/com/invisibleteam/goinvisible/mvvm/edition/EditActivity.java]: Please camelCase

**Review by BartSoftware — CHANGES_REQUESTED** [review]

**Review by BartSoftware — APPROVED** [review]

**Review by michal1990 — APPROVED** [review]

**Pull request merged**